addons/mail/static/src/core/common/message_model.js

```diff
@@ -462,28 +462,47 @@ export class Message extends Record {
     previewText = fields.Html("", {
         /** @this {import("models").Message} */
         compute() {
+            let messageBody = "";
             if (!this.hasOnlyAttachments) {
-                return this.inlineBody || this.subtype_id?.description;
+                messageBody = this.inlineBody || this.subtype_id?.description;
+            } else {
+                const attachments = this.attachment_ids;
+                switch (attachments.length) {
+                    case 1:
+                        messageBody = attachments[0].previewName;
+                        break;
+                    case 2:
+                        messageBody = _t("%(file1)s and %(file2)s", {
+                            file1: attachments[0].previewName,
+                            file2: attachments[1].previewName,
+                            count: attachments.length - 1,
+                        });
+                        break;
+                    default:
+                        messageBody = _t("%(file1)s and %(count)s other attachments", {
+                            file1: attachments[0].previewName,
+                            count: attachments.length - 1,
+                        });
+                }
             }
-            const { attachment_ids: attachments } = this;
-            if (!attachments || attachments.length === 0) {
-                return "";
-            }
-            switch (attachments.length) {
-                case 1:
-                    return attachments[0].previewName;
-                case 2:
-                    return _t("%(file1)s and %(file2)s", {
-                        file1: attachments[0].previewName,
-                        file2: attachments[1].previewName,
-                        count: attachments.length - 1,
-                    });
-                default:
-                    return _t("%(file1)s and %(count)s other attachments", {
-                        file1: attachments[0].previewName,
-                        count: attachments.length - 1,
-                    });
+
+            if (this.isSelfAuthored) {
+                return markup(
+                    '<i class="fa fa-mail-reply me-1 opacity-75"/>' +
+                        _t("You: %(messageBody)s", {
+                            messageBody,
```

Comment:
`<i class=""/>` syntax is invalid HTML, and it actually puts `messageBody` inside `<i>` as a result:
<img width="1280" height="1117" alt="Image" src="https://github.com/user-attachments/assets/eb61394e-11f6-48ff-8c93-3ccaa472ec07" />

I would advice to `renderToMarkup`. AFAIK all locales do have `X: Y` format, this looks overkill to allow reorder.